packages/office-ui-fabric-react/src/components/DetailsList/DetailsHeader.base.tsx

```diff
@@ -267,7 +267,7 @@ export class DetailsHeaderBase extends BaseComponent<IDetailsHeaderBaseProps, ID
           : null}
         {groupNestingDepth! > 0 && this.props.collapseAllVisibility === CollapseAllVisibility.visible ? (
           <div className={classNames.cellIsGroupExpander} onClick={this._onToggleCollapseAll} data-is-focusable={true}>
-            <Icon className={classNames.collapseButton} iconName="ChevronDown" />
+            <Icon className={classNames.collapseButton} iconName={isRTL ? 'ChevronLeftMed' : 'ChevronRightMed'} />

```

Comment:
Wow.. nice catch @Vitalius1!

Reply:
Interesting.. I guess this could _also_ be done by keeping `ChevronDown` but adjusting the styles applied based on `getRTL();` 🤔

Reply:
I talked to Ben about just styling the ChevronDown, but the Icons I replaced it with are also a little thicker (notice the termination Med). So we decided to go with the replacement.

Reply:
It looks real nice and animates correctly in RTL / LTR, so you have my vote!

Reply:
I think we should communicate this change overall so folks can apply custom style overrides and the existing icon when they upgrade. No surprises.

Reply:
Yeah, agree that we need to communicate this change. What would be the best way to do it? I already have a short mention of this in the change log. Should I try and emphasize it more?